src/storage/src/monitor/local_metrics.rs

```diff
@@ -73,78 +74,126 @@ impl StoreLocalStatistic {
         self.cache_meta_block_miss += local_cache_meta_block_miss;
     }
 
-    pub fn report(&self, metrics: &StateStoreMetrics) {
+    pub fn report(&self, metrics: &HummockStateStoreMetrics, table_id_label: Option<&str>) {
```

Comment:
table_id_label can just ba `&str` and we can use a empty string for use cases without a table id